src/components/menu/scrollable-block/scrollable-block.spec.tsx

```diff
@@ -13,19 +13,31 @@ import { assertStyleMatch } from "../../../__spec_helper__/test-utils";
 
 const handleKeyDownFn = jest.fn();
 
-const submenuContextValues = (blockIndex, focusIndex) => ({
+const submenuContextValues = (blockIndex?: number, focusIndex?: number) => ({
   handleKeyDown: handleKeyDownFn,
   blockIndex,
   focusIndex,
 });
 
 describe("ScrollableBlock", () => {
-  let container;
-  let wrapper;
-
-  const render = (menuType, props, blockIndex, focusIndex) => {
+  let container: HTMLDivElement | null;
+  let wrapper: ReactWrapper;
+
+  const render = (
+    menuType: MenuType,
+    props = {},
```

Comment:
Similar to previous comments
```suggestion
    props: Partial<ScrollableBlockProps> = {},
```